docker/Dockerfile-api

```diff
@@ -26,8 +26,15 @@ RUN pip3 install -r requirements.txt --no-deps --no-cache-dir --require-hashes \
 # Use the system CA bundle for the requests library
 ENV REQUESTS_CA_BUNDLE=/etc/pki/tls/certs/ca-bundle.crt
 
+# Temp dir used to collate the metrics between different threads by
+# the Prometheus Flask Exporter
+RUN mkdir /tmp/prometheus-flask-metrics
+
```

Comment:
The `init_metrics()` function will create the directory if it doesn't exist, right? So this is not needed?

Reply:
Yes.  Removed.